src/preact/base-element.js

```diff
@@ -260,11 +274,15 @@ export class PreactBaseElement extends AMP.BaseElement {
     const Ctor = this.constructor;
 
     this.observer = new MutationObserver(this.checkMutations_.bind(this));
-    const childrenInit = Ctor['children'] ? CHILDREN_MUTATION_INIT : null;
-    const passthroughInit =
-      Ctor['passthrough'] || Ctor['passthroughNonEmpty']
-        ? PASSTHROUGH_MUTATION_INIT
-        : null;
+    const props = Object.values(Ctor['props']);
+    const childrenInit = props.some((p) => p.selector)
```

Comment:
Rare for us, but let's instruct both of these `some()` functions as top level functions. A small optimization, but they are completely free of dependencies and we can avoid create new functions all the time.

Reply:
Done for the `some()` and `every()` calls, PTAL and let me know if this is what you had in mind?